geniza/corpus/templates/admin/corpus/document/change_form.html

```diff
@@ -0,0 +1,15 @@
+{% extends "admin/change_form.html" %}
+{% block after_field_sets %}
+<fieldset class="module aligned">
+    <div class="form-row">
+        <div class="fieldBox">
+            <label>First input:</label>
+            {% include "admin/corpus/document/rev_log_entry.html" with log_entry=first_input %}
+        </div>
+        <div class="fieldBox">
+            <label>Last revision:</label>
+            {% include "admin/corpus/document/rev_log_entry.html" with log_entry=last_revision %}
+        </div>
+    </div>
```

Comment:
Can we construct a link to the record history? I know it's in the top right, but seems like it would be convenient to add here (something like "view full history")

Reply:
here's the list of admin url patterns, which includes history: https://docs.djangoproject.com/en/3.1/ref/contrib/admin/#reversing-admin-urls

Reply:
I'm trying to figure out a way to make this less visually awkward. would it be with the text of one of the fields? It almost seems easier to just move these fields to the top so they're next to the history button.

Reply:
Can you share a screenshot to show how it's awkward?

If it doesn't make sense we don't need the history link, just thought it would be convenient.

Reply:
is this what you were imagining?

<img width="685" alt="Screen Shot 2021-04-14 at 5 35 28 PM" src="https://user-images.githubusercontent.com/4924494/114782805-d95e3200-9d47-11eb-85b6-d126029ac897.png">

Reply:
that looks reasonable to me